trymito.io/pages/use-cases/python-training.tsx

```diff
@@ -0,0 +1,320 @@
+                            </div>
+                            <h3>
+                                Decrease errors by 8x
+                            </h3>
+                            <p>
+                                No more syntax errors or forgetting a colon. Mito generates code for you, so your analysts can focus on the business logic. 
+                            </p>
+                            <p>
+                                Help your analysts leapfrog the <a className={pageStyles.link} href='https://www.trymito.io/blog/10-mistakes-to-look-out-for-when-transitioning-from-excel-to-python' target="_blank" rel="noreferrer">“Oh my god. Why is this not working? UGGHHH!!!”</a> phase. 
+                            </p>
+                        </div>
+                    </div>
+                    <div className={pageStyles.subsection}>
+                        <div className={securityStyles.security_bullet_container}>
+                            <div className={securityStyles.icon}>
+                                <Image  src={dataIconPurple} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Improve Data Discoverability
+                            </h3>
+                            <p>
+                                Figuring out which database holds the monthly report&apos;s data is often a multi-week scavenger hunt. Then getting permission to query that database might be another few weeks of waiting.
+                            </p>
+                            <p>
+                               Pre-load common SQL queries into Mito so your business analysts can start working with their data immediately.
+                            </p>
+                            <p>
+                                To get the most out of your large data, the subject matter expert needs to be the one analyzing it. 
+                                Not a supporting data scientist who isn&apos;t using the data to make business decisions.
+                            </p>
+                        </div>
+                        <div className={pageStyles.subsection_second_element_mobile_spacing}>
+                            <div className={securityStyles.icon}>
+                                <Image className={securityStyles.icon} src={infoIcon} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Skip pandas syntax 
+                            </h3>
+                            <p>
+                                Since Mito generates the equivalent Python code for every edit, you don&apos;t need to teach confusing pandas syntax.
+                            </p>
+                            <p>
+                                Test yourself. Which of these is correct?
+                            </p>
+                            <ol style={{'listStylePosition': 'inside', 'paddingLeft': '0'}}>
```

Comment:
It feels like these should be clickable, as every other quiz is on early with multiple choice :) 

Maybe:
1. Highlight on hover
2. Just highlight the correct answer in purple when they click